### DISCUSSION TITLE
Help the DEI SIG Choose a Name for the DEI Events Series!

### DISCUSSION THREAD

**Discussion by eteridvalishvili** [finos/DEI-SIG#73] [Polls]
As the DEI SIG prepares to launch this exciting new series of DEI-focused events, the SIG needs your input to select the perfect name! Below are the proposed options. Please vote by choosing your top choice below.

**Comment by ojeb2** [comment]
Surely **FinOS #include** ?

**Reply by alvin-c-shih** [reply]
Can self-identify as old by saying "number include" and as young by saying "hashtag include". :-)

**Comment by mphomathabathe** [comment]
FINOS Collective 🚀

**Reply by DamianReeves** [reply]
With you on FINOS Collective

**Comment by sammoorhouse** [comment]
Someone has suggested "FINOS Bigger Table", which I really love. Please give this comment a thumbs-up if you agree.

**Reply by alvin-c-shih** [reply]
Sounds like the start of a rivalry...?
- https://cloud.google.com/bigtable

**Comment by amysak-finos** [comment]
I am not sure I like any of these. They are all a bit of a mouthful. Slogan/name should be easy to say - maybe simply "FINOS FOR ALL"
